front/app/containers/Admin/projects/project/analysis/Heatmap/SummarizeButton.tsx

```diff
@@ -144,6 +144,13 @@ const SummarizeButton = ({ row, column, cell, buttonStyle }: Props) => {
 
   const isDisabled = !cell || column.relationships.custom_field_option === null;
 
+  // Only show the button if the cell is of type 'inputs'
+  // This is the only filter we fully support for now so showing the button
+  // for other types would be misleading
+  if (cell?.attributes.unit !== 'inputs') {
```

Comment:
@kogre This hides the summarize button for the other types of unit. I had this logic before but it got lost in the refactor so bringing it back now to avoid confusion.